x-pack/plugins/security/server/authentication/authenticator.ts

```diff
@@ -728,9 +731,14 @@ export class Authenticator {
    * Invalidates session value associated with the specified request.
    * @param request Request instance.
    * @param sessionValue Value of the existing session if any.
+   * @param skipAuditEvent If enabled, skips writing a `user_logout` audit event for this session
    */
-  private async invalidateSessionValue(request: KibanaRequest, sessionValue: SessionValue | null) {
-    if (sessionValue) {
+  private async invalidateSessionValue(
+    request: KibanaRequest,
+    sessionValue: SessionValue | null,
+    skipAuditEvent?: boolean
```

Comment:
__nit:__ Since we've got a boolean optional param, how do you feel about grouping all these three parameters into a params/options object? This way we'll have a more readable code on the consumer side (boolean positional params are always hard to "parse", infamous [`url.parse`](https://nodejs.org/api/url.html#urlparseurlstring-parsequerystring-slashesdenotehost) comes to my mind).
```ts
/**
 */
interface InvalidateSessionParams {
  request: KibanaRequest;
  sessionValue: SessionValue | null;
  /**
   * Nice JSDoc written only in a single place :)
   */
  skipAuditEvent?: boolean;
}
```

Reply:
Great suggestion, fixed in 838c3be!